formulas/states/saline-prometheus/init.sls

```diff
@@ -4,7 +4,7 @@ saline-prometheus-cfg:
   - name: /etc/prometheus/saline.yml
   - contents: |
       - targets:
-        - {{ salt['pillar.get']('mgr_server') }}:{{ salt['pillar.get']('prometheus:saline_port') }}
+        - {{ salt['pillar.get']('mgr_server') }}/saline }}
 {%- if salt['pillar.get']('prometheus:saline_https_connection', False) %}
         labels:
           __scheme__: https
```

Comment:
For some reason I can't make working the config from your PR, maybe I missed something important, but the suggested below works fine for me:
```suggestion
        - {{ salt['pillar.get']('mgr_server') }} }}
        labels:
          __metrics_path__: /saline/metrics
{%- if salt['pillar.get']('prometheus:saline_https_connection', False) %}
          __scheme__: https
```

Reply:
I've seen that in my tests and your suggestion is better than my solution